tests/google-cloud-python-system/Dockerfile.in

```diff
@@ -0,0 +1,16 @@
+FROM ${FULL_BASE_IMAGE}
+
+# Get the source.
+RUN git clone --depth 1 https://github.com/GoogleCloudPlatform/google-cloud-python.git
+WORKDIR google-cloud-python
+
+# Install nox
+RUN pip install --upgrade nox-automation
+
+# Secrets injected at runtime
+ENV GOOGLE_APPLICATION_CREDENTIALS=/workspace/tests/google-cloud-python-system/credentials.json
```

Comment:
FYI, this will be unnecessary (and we shouldn't do it) once the builder robot scopes issue is fixed, assuming the build is kicked off from an authenticated host machine. It's the same issue that's preventing me from hooking up the integration tests in cloudbuilds right now, but should be fixed very soon™.